src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -572,7 +572,9 @@ static void ParseCommandLineFeatureFlags()
 					{
 						try
 						{
-							property.SetValue(null, Convert.ChangeType(value, property.PropertyType));
+							// ChangeType doesn't handle Nullable types
```

Comment:
For future reference, this can be removed if https://github.com/dotnet/runtime/issues/96216 is ever implemented.